src/Orc.Controls/Orc.Controls.Shared/Controls/DatePicker/Views/DatePickerControl.xaml

```diff
@@ -4,10 +4,15 @@
                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
                    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
                    xmlns:controls="clr-namespace:Orc.Controls"
+                   xmlns:converters="clr-namespace:Orc.Controls.Converters"
                    xmlns:catel="http://catel.codeplex.com"
                    xmlns:sys="clr-namespace:System;assembly=mscorlib"
                    mc:Ignorable="d" x:Name="CurrentControl" d:DesignHeight="22" d:DesignWidth="190">
 
+    <catel:UserControl.Resources>
+        <converters:YearLongToYearShortConverter x:Key="YearConverter" />
```

Comment:
I prefer to use the same key as the name of the converter (for example, we have BooleanToHidingVisibilityConverter and BooleanToCollapsingVisibilityConverter). We need to be consistent, so please use 'YearLongToYearShortConverter'  here.